cmd/podman/rmi.go

```diff
@@ -70,12 +71,16 @@ func rmiCmd(c *cli.Context) error {
 			imagesToDelete = append(imagesToDelete, newImage)
 		}
 	}
+
 	if len(imagesToDelete) == 0 {
 		return errors.Errorf("no valid images to delete")
 	}
 	for _, img := range imagesToDelete {
 		msg, err := runtime.RemoveImage(img, c.Bool("force"))
 		if err != nil {
+			if errors.Cause(err) == storage.ErrImageUsedByContainer {
+				fmt.Printf("A container associated with Buildah may be associated with this image.\n")
```

Comment:
Shouldn't just call out buildah here - could also be CRI-O.

Reply:
Ideally, should mention containers/storage specifically, could be a program using c/storage we didn't write

Reply:
Couple of things here.  First, if the err is no ErrImageUserByContainer, isn't the error lost?  And second, should your error message also contain the contents of err?

Reply:
I've changed the error message, bumped it a bit from what @mheon suggested.  I don't think the error is lost here, we save it as we do currently in lastError.  This only prints out extra verbiage when we have a container tied to the image.  The error that prints I didn't think added a lot here, and the error message is currently pretty long so I didn't include it.  If you've a strong opinion on adding it I can.